bool flag
int16 cls
float32 conf 